docx/oxml/text/paragraph.py

```diff
@@ -83,13 +84,13 @@ def style(self, style):
     def r_lst_recursive(self):
         """
         Override xmlchemy generated list of runs to include runs from
-        hyperlinks.
+        hyperlinks and content controls.
```

Comment:
When you are there you could change `r_lst_recursive` to `iter_runs_recursive`. It looks like the name would be more consistent with the rest of the code.